src/_actions/WorkspaceActions.js

```diff
@@ -22,10 +22,15 @@ export const updateTickTradeSubmarket = submarket => ({
     submarket,
 });
 
-export const updateTickTradeDate = date => ({
-    type: types.UPDATE_TRADING_TIMES_DATE,
-    date,
-});
+export const updateTradingTimesDate = date => {
```

Comment:
A request for updating the trade times does not need to be an action, right?
It is not changing the state in itself, the incoming data from server is calling the update for the state.